docs/dxp/7.x/en/installation-and-upgrades/setting-up-liferay-dxp/clustering-for-high-availability/example-creating-a-dxp-cluster.md

```diff
@@ -81,37 +111,65 @@ Your database server is ready for DXP.
 
 A DXP cluster requires a search engine server that's accessible to all of the app server nodes. See [Configuring a Search Engine in a Cluster](./clustering-search.md) for more information.
 
-Create a search engine server container based on an [Elasticsearch Docker image](https://hub.docker.com/_/elasticsearch):
+Create a search engine server container based on a [Elasticsearch Docker image](https://hub.docker.com/_/elasticsearch):
```

Comment:
Shouldn't this be _**an** Elasticsearch Docker image_?

Reply:
yeah now that I re-read it, you're right.